packages/core/src/Avatar/Avatar.tsx

```diff
@@ -168,7 +168,13 @@ export const HvAvatar = forwardRef<
       <MuiAvatar
         component={component}
         // Consider not using the root and className classes in this component
-        className={cx(classes.root, classes.avatar, classes[size], className)}
+        className={cx(
+          classes.root,
+          classes.avatar,
+          classes[size],
+          classes[variant],
```

Comment:
```suggestion
```

We shouldn't duplicate styles in _different_ components. Also, the "variants" should be in the root, so that we can use a selector from anywhere:

```ts
classes: {
  square: {
    "& .HvAvatar-avatar": {
      borderRadius: 0
    }
  }
}